opencog/haskell/LICENSE

```diff
@@ -0,0 +1,693 @@
+Unless indicated otherwise (a license reference in file headers, or a 
+LICENSE file in a directory), all OpenCog source files use the AGPL 
+with additional permissions to link OpenCog libraries with non-AGPL
+works; the text of the OpenCog linking excpetion is located at the end
+of this file.
+
+ 
+
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
Don't include a separate license file. The master license file in the main directory covers everything, already.